newsletter-november-2022.rst

```diff
@@ -11,34 +11,39 @@
 Read the Docs newsletter - November 2022
 ========================================
 
-A bunch of new features and updates have emerged since we announced a Q4 focus on core platform features in the :doc:`previous newsletter <newsletter-october-2022>`.
+Here are the first features and updates that have hatched since we announced a Q4 focus on core platform features in the :doc:`previous newsletter </newsletter-october-2022>`.
 
 
 New features
 ------------
 
 The latest updates from our team:
 
-- :doc:`sphinx-rtd-theme 1.1.0 <theme-release-110>` has been released.
-- After rolling out auto-cancelling builds `last month <https://blog.readthedocs.com/cancel-old-builds/>`_, we have alread recorded a whopping 10% decrease in builds.
+- :doc:`sphinx-rtd-theme 1.1.0 </theme-release-110>` has been released with minor improvements and as a step towards more development in the near future.
+- After rolling out auto-cancelling builds `last month <https://blog.readthedocs.com/cancel-old-builds/>`_, we have already recorded a whopping 10% decrease in builds.
   We're really happy with how this turned out, not least that it effectively reduces our cloud footprint by 10% 🌱.
 - We started incrementally refactoring our own documentation to match the `Diátaxis framework <https://diataxis.fr/>`_. We will be writing more about this, as we are currently gaining practical experience.
-- Much thanks to `AA-Turner <https://github.com/AA-Turner>`_ and the Sphinx community for working together on a proposal and releasing the extension `sphinxcontrib-jquery <http://pypi.org/project/sphinxcontrib-jquery>`_ which themes and extensions that need jQuery will have to add as a dependency once Sphinx 6 is released.
+- Much thanks to `AA-Turner <https://github.com/AA-Turner>`_ and the Sphinx community for working together on a proposal and releasing the extension `sphinxcontrib-jquery <http://pypi.org/project/sphinxcontrib-jquery>`_. This extension is required for themes and extensions that need jQuery from Sphinx 6 and onewards. Sphinx 6.0 is scheduled for December 2022.
 
 You can always see the latest changes to our platforms in our :doc:`Read the Docs Changelog <readthedocs:changelog>`.
 
+
 Upcoming features
 -----------------
 
+- Soon, a 1.2.0 release of :ref:`sphinx-rtd-theme <sphinx_rtd_theme110_upcoming_releases>` will support docutils 0.18 and Sphinx 6.
```

Comment:
🍿

Reply:
Yup..

Reply:
Let me know if there's anything you find to bold about this announcement -- I just find it necessary to remind people that run into these issues that we are committed to fixing them. docutils 0.18 support is on a lot of peoples' radar AFAICT: